src/gettingstarted/own-code/connect-services.md

```diff
@@ -0,0 +1,157 @@
+# Import your own code
+
+## Connect to Services
+
+> **Note:** If your application does not require access to services and you left `.platform/services.yaml` blank, feel free to proceed to the next step.
+
+At this point you have configured your application's services as well as its build and deploy process in your `.platform/services.yaml` and `.platform.app.yaml` files. As an example, in your `.platform.app.yaml` file you may have defined a relationship called `database`:
+
+```yaml
+relationships:
+    database: "mysqldb:mysql"
+```
+
+which was configured in `.platform/services.yaml` with
+
+```yaml
+mysqldb:
+    type: mysql:10.2
+    disk: 1024
+```
+
+In order to connect to this service and use it in your application, Platform.sh exposes its credentials in the application container within a base64-encoded JSON `PLATFORM_RELATIONSHIPS` environment variable.
+
+To access this variable you can install a Platform.sh configuration reader library
+
+{% codetabs name="PHP", type="php" -%}
```

Comment:
The `type` for these should be `bash`, I think, because the content is a command line statement. (Except Java and maybe Go.  Should Java be 2 tabs, one for Maven (type XML) and one for Gradle (type ihavenoidea)?